src/components/ErrorsContainer.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component, PropTypes } from 'react';
+import FieldConnect from './FieldConnect';
+import ErrorField from './ErrorField';
+
+const parseErrors = (data) => {
```

Comment:
Please work on complexity. Now if I good see we have complexity: 7. Please move some logic to seperate functions. Complexity: 4 should be max for one function.